cekit/cli.py

```diff
@@ -312,14 +312,16 @@ def test(image, overrides):
     show_default=True,
 )
 @click.option("--wip", help="Run test scenarios tagged with @wip only.", is_flag=True)
+@click.option("--include-re", help="Run only those features which match this regex")
+@click.option("--exclude-re", help="Do not run those features which match this regex")
```

Comment:
Can we add these parameters to the user-guide (in file `docs/handbook/testing/behave.rst` ) please?

Reply:
Documentation added!